lib/tether/tether.go

```diff
@@ -236,12 +236,17 @@ func (t *tether) setNetworks() error {
 
 func (t *tether) setMounts() error {
 	for k, v := range t.config.Mounts {
-		if v.Source.Scheme != "label" {
+		switch v.Source.Scheme {
+		case "label":
+			// this could block indefinitely while waiting for a volume to present
+			t.ops.MountLabel(context.Background(), v.Source.Path, v.Path)
+
+		case "nfs":
+			t.ops.MountFileSystem(context.Background(), v.Source, v.Path, v.Mode)
```

Comment:
I mean it does need some sort of encoding I am sure, But I am not sure if the raw syscall will take care of that or not. There isn't a provided encoding that I know of for nfs.